sigs.yaml

```diff
@@ -3295,6 +3295,11 @@ sigs:
       name: Petr Muller
       company: Red Hat
       email: muller@redhat.com
+  - name: randfill
+    description: |
+      Utility functions for filling types with randomized data (for kubernetes testing use).
+    owners:
+    - https://raw.githubusercontent.com/kubernetes-sigs/randfill/main/OWNERS
```

Comment:
[non-blocking, could be a follow-up]

I suppose let's start by listing @thockin under subprojects leads here?

Reply:
SGTM

Reply:
updated the commit. PTAL. Thanks!